src/shared/__tests__/actions/transfers.spec.js

```diff
@@ -412,7 +412,7 @@ describe('actions: transfers', () => {
                         .dispatch(
                             actions.makeTransaction(
                                 seedStore,
-                                'UUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUNXELTUENX',
+                                'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYLFHUOJUY',
```

Comment:
Since addresses ending in letters E-V cannot be used, value transfer tests for `makeTransaction` started failing